Classes/Driver/Version1/DocumentDriver.php

```diff
@@ -28,14 +28,21 @@
 class DocumentDriver extends AbstractDriver implements DocumentDriverInterface
 {
     /**
+     * @Flow\Inject
+     * @var NodeTypeMappingBuilderInterface
+     */
+    protected $nodeTypeMappingBuilder;
+
+    /**
      * {@inheritdoc}
      */
     public function delete(NodeInterface $node, $identifier)
     {
+        $nodeTypeMappingBuilder = $this->nodeTypeMappingBuilder;
```

Comment:
Because `$this->nodeTypeMappingBuilder::convertNodeTypeNameToMappingName($node->getNodeType())` leads to a syntax error in PHP < 7, see https://3v4l.org/Hf8aj